numpy/random/mtrand/mtrand.pyx

```diff
@@ -1732,17 +1749,17 @@ cdef class RandomState:
         """
         if high is None:
             warnings.warn(("This function is deprecated. Please call "
-                           "randint(1, {low} + 1) instead".format(low=low)),
+                           "randint(low=1, high={low} + 1) instead".format(low=low)),
```

Comment:
Well the deprecation _technically_ has not been released in an official version yet, and I also see no reason why people shouldn't be encouraged to use a better style when passing in arguments to essentially an all `kwargs` function.